build_all/base-libraries/AzureIoTHub/README.md

```diff
@@ -66,22 +66,21 @@ You should have the following ready before beginning with any board:
 
 5. Navigate to where your esp8266 board package is located, typically in `C:\Users\<your username>\AppData\Local\Arduino15\packages` on Windows and `~/.arduino15/packages/` on Linux
 	
-- Locate the board's Arduino.h (`hardware/esp8266/<board package version>/cores/esp8266/` and comment out the line containing `#define round(x)`, around line 137.
+- Locate the board's `Arduino.h` (`hardware/esp8266/<board package version>/cores/esp8266/` and comment out the line containing `#define round(x)`, around line 137.
 
-- Locate the board's `platform.txt` and add the defines `-DDONT_USE_UPLOADTOBLOB` `-DUSE_BALTIMORE_CERT` on line 73 (`build.extra_flags=`) 
+- Two folders up from the `Arduino.h` step above, in the same folder as the board's `platform.txt`, copy the `platform.local.esp8266.txt` file to it and rename the file `platform.local.txt` 
```

Comment:
same comment as below about moving the different versions to subdirectories.

Reply:
Done.